src/app/detail/ui/TaskActivityComponent.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import { Box, Stack, Typography } from '@mui/material'
+import { ActivityLog } from './ActivityLog'
+
+interface Prop {
+  activities: any
+}
+
+export const TaskActivityComponent = ({ activities }: Prop) => {
+  return (
+    <>
+      <Stack direction="column" alignItems="left" p="10px 5px" rowGap={5}>
+        <Typography variant="xl">Activity</Typography>
+        <Stack direction="column" alignItems="left" p="10px 5px" rowGap={4}>
+          {activities.map((item: any, index: number) => {
+            return (
+              <Box
+                sx={{
+                  height: 'auto',
+                  display: 'block',
+                }}
+                key={item.id}
+              >
+                <ActivityLog log={item} isLast={index == activities.length - 1} />
+              </Box>
+            )
+          })}
+        </Stack>
+      </Stack>
+    </>
+  )
+}
```

Comment:
This component shouldn't have any state or events. I believe it is just about rendering the static content. Then why use `'use client'` and bring this on the client side to render? We can render this in the server side as activity log is a static content. 

We also don't need separate components such as `ActivityLog.tsx` and `TaskActivityComponent.tsx` just to render the list of activity logs. This is creating unnecessary modularization which is confusing. We can directly make use of single `server` component to handle the whole activity log (not comment) related rendering.